Makefile.am

```diff
@@ -80,15 +80,7 @@ EXTRA_DIST = $(SAFEC_INFRA) $(SAFEC_GENHEADERS) $(SAFEC_SLKM)
 
 BUILT_SOURCES = $(top_srcdir)/.version
 
-# Install the generated pkg-config file (.pc) into the expected location for
-# architecture-dependent package configuration information.  Occasionally,
-# pkg-config files are also used for architecture-independent data packages,
-# in which case the correct install location would be $(datadir)/pkgconfig.
-#      pkgconfigdir = $(libdir)/pkgconfig
-#      pkgconfig_DATA = safec-$(SAFEC_API_VERSION).pc
-# We need to be architecture-dependent:
-pcdatadir = $(libdir)/pkgconfig
-pcdata_DATA = safec-$(SAFEC_API_VERSION).pc
+pkgconfig_DATA = libsafec.pc
```

Comment:
nope. we decided long time ago to strongly version libsafec.

Reply:
What was the impetus behind that decision?  It makes it difficult to use programmatically in a generic fashion from a third-party application's build system since it must always be tied to a particular version of the library.  For context, Snort is currently using libsafec-1.0 and I was trying to update it to use your revived project, but I don't really want to have to rev Snort to look for the new pkg-config name every time you touch the API version.  How often are you planning to rev the API version?  Will it rev even if the changes were only additive?  Also, the headers are not currently being installed into versioned include directories (see the changes in libsafec.pc.in to reflect this) which would be necessary for the parallel installations the strong versioning would require.

Reply:
Well, the version bump happens every major and minor release. So you can rely on that very particular API. If you need the latest API, you have to change your build setup. That was the idea. They didn't do that with the headers (like glib), just the lib.
This was done with the 2nd commit 8379e6f6e74afec9e96eae41853567faec6fb273 2012.

Reply:
Sorry, I've been sort of swamped and haven't circled back to this.  If you really want to keep strictly versioning every aspect of libsafec, let me rework this PR to handle that.  I assume that Ubuntu 14.04 support is a hard requirement (for the pkg-config thing)?

Reply:
I've added a ticket for unversioning it: #56. 
I'm leaning towards variant 3, add an unversioned .pc and install both.